source/guides/core-concepts/variables-and-aliases.md

```diff
@@ -7,24 +7,24 @@ title: Variables and Aliases
 # {% fa fa-graduation-cap %} What You'll Learn
 
 - How to deal with async commands
-- What Aliases are, how they simplify your code
+- What Aliases are and how they simplify your code
 - Why you rarely need to use variables with Cypress
 - How to use Aliases for objects, elements and routes
 {% endnote %}
 
 # Return Values
 
-New users to Cypress may initially find it challenging to work with the asynchronous nature of our API's.
+New users to Cypress may initially find it challenging to work with the asynchronous nature of our APIs.
 
 {% note success 'Do not worry!' %}
 There are many simple and easy ways to reference, compare and utilize the objects that Cypress commands yield you.
 
-Once you get the hang of async code - you'll realize you can do everything you could do synchronously, without your code doing any backflips.
+Once you get the hang of async code you'll realize you can do everything you could do synchronously, without your code doing any backflips.
```

Comment:
Corrected a few sentences where ` - ` is used to indicate a clause or pause (I think). The sentences read just fine if it is removed though.... so I removed it. Style thing? No biggie.